tracer/src/Datadog.Trace/CorrelationIdentifier.cs

```diff
@@ -59,14 +59,26 @@ public static string Env
         }
 
         /// <summary>
-        /// Gets the id of the active trace.
+        /// Gets the 128-bit id of the active trace as a hexadecimal string.
         /// </summary>
-        /// <returns>The id of the active trace. If there is no active trace, returns zero.</returns>
+        /// <returns>The id of the active trace. If there is no active trace, returns the empty string.</returns>
+        public static string TraceId128
```

Comment:
I saw you said we should expose it to users. To better understand, why do we need to?

Reply:
+1 is this necessary? Could we defer this until we see a specific use case for it?

Reply:
For the same reasons we already expose the trace id and span id? Used can use it to manually inject headers or into logs. If they enable 128-bit trace ids, they will access to it without truncating it to 64 bits.

Reply:
Removed after discussion with team.